project_leverage/controllers/main.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (C) 2020 Open Source Integrators
+# Copyright (C) 2020 Serpent Consulting Services
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.http import request
+from odoo.exceptions import UserError
+
+from odoo.addons.sale_timesheet.controllers.main import SaleTimesheetController
+
+
+DEFAULT_MONTH_RANGE = 3
+
+
+class SaleTimesheetController(SaleTimesheetController):
+
+    def _plan_prepare_values(self, projects):
+
+        employee_obj = request.env['hr.employee']
+        values = super(SaleTimesheetController,
+                       self)._plan_prepare_values(projects)
+        _select = self.select_line()
+        _from = self.line_from()
+        _where = self.line_where(projects.ids)
```

Comment:
You're passing an IDs list, to later browse it into a Rset again.
```suggestion
        _where = self.line_where(projects)
```